docs/sample_log_config.yaml

```diff
@@ -28,14 +28,17 @@ handlers:
     # will be a delay for INFO/DEBUG logs to get written, but WARNING/ERROR
```

Comment:
this isn't quite accurate any more.

Reply:
I see what you mean, but it also *kind* of is. I have updated the wording now, mind letting me know if you think it's better?